### PR TITLE
FEAT: added homepage

### PR DESCRIPTION
**Pull request by zoyatanweer**
files to check:
- Homepage.jsx
- Header.jsx

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              >
+                <LogOutIcon />
+                Logout
+              </button>
+            </NavLink>
+          ) : (
+            <NavLink to="/login">
+              <button class="btn box-shadow btn-primary">Login</button>
+            </NavLink>
+          )} */}
```
Comment (Shubhamkr2610) [inline src/Components/Header/Header.jsx:58]: if the code is for future use , you can mention in comment , else remove it

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          <NavLink to="/">
+            Falcon <span className="title-theme-name">Pomodoro</span>
+          </NavLink>
+        </div>
+        {/* <div className="searchbar">
+          <input
+            className="desktop-searchbar"
+            type="text"
+            placeholder="Search for videos"
+          />
```
Comment (Shubhamkr2610) [inline src/Components/Header/Header.jsx:37]: if this code is for future use , you can mention in comment , else remove it

File: src/Components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.navigation-container {
+  background-color: var(--background-color);
+  margin: 1rem;
+}
+.nav-brand .title-theme {
+  color: black;
+}
+.title-theme-name {
+  color: #f9c543;
+}
```
Comment (Shubhamkr2610) [inline src/Components/Header/Header.css:10]: use color with variable name

**Review by srishtigautam21 — APPROVED** [review]
The code clean 
Good to merge

File: src/Pages/Homepage/Homepage.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+.para-medium {
+  font-size: 4vh;
+}
+
+.btn-primary {
+  background-color: var(--yellow-color);
+}
+.btn-primary:hover {
+  background-color: #f1b104;
+}
```
Comment (Shubhamkr2610) [inline src/Pages/Homepage/Homepage.css:28]: use color with variable name to maintain consistency

**Review by Shubhamkr2610 — APPROVED** [review]
Suggestion :  After removing  unused code , you can merge

**Pull request merged**